swagger.yml

```diff
@@ -80,10 +80,24 @@ paths:
           schema:
             $ref: "#/definitions/MeetingRoomInfo"
 
+  /api/room/{roomId}:
```

Comment:
This endpoint would replace /api/room/{roomName}, so we should delete lines 69-81.